homework_postgres/PostgreSQL-Students.ipynb

```diff
@@ -40,9 +40,22 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    '''DELETE FROM roles WHERE movie_id = 1;
</pre><p>Delete join table can be construct in one query, ref: https://stackoverflow.com/questions/16481379/how-can-i-delete-using-inner-join-with-sql-server</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/haicheviet/dataeng-course-1/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='144'/>